src/Renci.SshNet/SftpClient.cs

```diff
@@ -1474,6 +1503,21 @@ public void Delete(string path)
             file.Delete();
         }
 
+        /// <summary>
+        /// Permanently deletes a file on remote machine.
+        /// </summary>
+        /// <param name="path">The name of the file or directory to be deleted. Wildcard characters are not supported.</param>
+        /// <param name="cancellationToken">The <see cref="CancellationToken"/> to observe.</param>
+        /// <returns>A <see cref="Task"/> that represents the asynchronous delete operation.</returns>
+        /// <returns>
+        /// A <see cref="Task"/> reprisenting the delete operation.
+        /// </returns>
```

Comment:
```suggestion
        /// <summary>
        /// Permanently deletes a file on the remote machine.
        /// </summary>
        /// <param name="path">The name of the file or directory to be deleted. Wildcard characters are not supported.</param>
        /// <param name="cancellationToken">The <see cref="CancellationToken"/> to observe.</param>
        /// <returns>A <see cref="Task"/> that represents the asynchronous delete operation.</returns>
```

And add to the interface